requirements.txt

```diff
@@ -32,7 +32,7 @@ testinfra==5.0.0
 jq==1.1.2 ; (platform_system == "Linux" or platform_system == "Darwin") and python_version <= "3.9"
 jq==1.2.2 ; (platform_system == "Linux" or platform_system == "Darwin") and python_version >= "3.10"
 cryptography==3.3.2; platform_system == "Linux" or platform_system == "Darwin" or platform_system=='Windows'
-urllib3>=1.26.5
+urllib3>=1.26.5,<2
```

Comment:
I would suggest usage of `urllib3<2`, this would install the latest non 2.x version.

Regarding the change itself it is not breaking as it is only used in two modules that are currently deprecated:
- qa_docs.py
- api_simulator.py